pkg/systemd/overview-cards/systemInformationCard.jsx

```diff
@@ -81,6 +92,43 @@ export class SystemInfomationCard extends React.Component {
                 });
     }
 
+    getSystemUptime() {
+        cockpit.spawn(["cat", "/proc/uptime"])
+                .then(text => {
+                    let uptime_days = 0;
+                    let uptime_hours = 0;
+                    let uptime_minutes = 0;
+
+                    const match = text.match(/[0-9]*\.[0-9]{2}/);
+                    let uptime_raw = match && parseFloat(match[0]);
+
+                    uptime_days = Math.floor(uptime_raw / 86400);
```

Comment:
Please don't reimplement time formatting, use [moment.duration](https://momentjs.com/docs/#/durations/) for that:

    this.setState({ systemUptime: moment.duration(uptime * 1000).humanize() })